pkg/ebpf/bytecode/asset_bindata.go

```diff
@@ -0,0 +1,17 @@
+// +build ebpf_bindata
+
+package bytecode
+
+import (
+	"bytes"
+)
+
+// GetReader returns a new AssetReader for the specified bundled asset
+func GetReader(dir, name string) (AssetReader, error) {
+	content, err := Asset(name)
+	if err != nil {
+		return nil, err
+	}
+
+	return bytes.NewReader(content), nil
+}
```

Comment:
Is it worth maintaining two builds with and without bindata? 

We're not going to distribute both, and it's not like users can switch between the two with a flag.

Reply:
IMHO yes. Otherwise testing internally would require an omnibus build, correct?

Reply:
For the runtime security agent, we are compiling the functional tests into a binary `go test -c -o`. It would help a lot to have everything in one binary

Reply:
Keeping the bindata also allows us to keep using the `TestEbpfBytesCorrect` to ensure the compiled versions are in sync with the sources

Reply:
@leeavital we deploy docker containers to clusters for testing quite often that don't have the source tree. If we don't have bundling, those Dockerfiles would need to be updated appropriately.